thinc/layers/with_nvtx_range.py

```diff
@@ -34,5 +32,6 @@ def backprop(dY: Any) -> Any:
     def init(_model: Model, X: Any, Y: Any) -> Model:
         return layer.initialize(X, Y)
 
-    return Model(f"nvtx_range({name})", forward, init=init, layers=[layer])
-
+    return Model(
+        f"nvtx_range({name})", forward, init=init, layers=[layer], shims=layer.shims
```

Comment:
Not sure if we want `shims=layers.shims` or `shims=[layers.shims[0]]` here?

Reply:
I think `layer.shims` is fine?

Reply:
I think `layer.shims` is fine?